lambda/audit.py

```diff
@@ -0,0 +1,319 @@
+                "repository": None,
+                "audit_id": audit_id,
+            }
+            LOG.info(event)
+    return True
+
+
+def log_org_teams(message=None):
+    """ Audit github organization teams """
+    org = os.environ.get("GITHUB_ORG")
+    audit_id = message.get("audit_id")
+    if audit_id is not None:
+        LOG.info({"action": "Audit organization teams", "org": org})
+        teams = github_api.get_github_org_teams(org)
+        event = {
+            "type": "OrganizationTeamCount",
+            "org": org,
+            "count": len(teams),
+            "audit_id": audit_id,
+        }
+        LOG.info(event)
+        for team in teams:
+            event = {
+                "type": "OrganizationTeam",
+                "org": org,
+                "member": None,
+                "team": team,
+                "repository": None,
+                "audit_id": audit_id,
+            }
+            LOG.info(event)
+            sns_team_membership = create_sns_message(
+                {
+                    "action": "log_org_team_membership",
+                    "org": org,
+                    "team": team,
+                    "audit_id": audit_id,
+                }
+            )
+            publish_alert(sns_team_membership)
+
+            sns_team_repos = create_sns_message(
+                {
+                    "action": "log_org_team_repos",
+                    "org": org,
+                    "team": team,
+                    "audit_id": audit_id,
+                }
+            )
+            publish_alert(sns_team_repos)
+    return True
+
+
+def log_org_team_membership(message=None):
+    """ Audit github organization team membership """
+    org = os.environ.get("GITHUB_ORG")
+    team = message.get("team", None)
+    team_name = team["slug"]
+    audit_id = message.get("audit_id")
+    if audit_id is not None:
+        LOG.info({"action": "Audit team members", "org": org, "team": team_name})
+        if team is not None:
+            members = github_api.get_github_org_team_members(org, team_name)
+            event = {
+                "type": "OrganizationTeamMemberCount",
+                "org": org,
+                "team": team,
+                "count": len(members),
+                "audit_id": audit_id,
+            }
+            LOG.info(event)
+            for member in members:
+                event = {
+                    "type": "OrganizationTeamMember",
+                    "org": org,
+                    "member": member,
+                    "team": team,
+                    "repository": None,
+                    "audit_id": audit_id,
+                }
+                LOG.info(event)
+        else:
+            LOG.error({"error": "Team not specified"})
+
+    return True
+
+
+def log_org_team_repos(message=None):
+    """ Audit github organization repo team access """
+    org = os.environ.get("GITHUB_ORG")
+    team = message.get("team", None)
```

Comment:
Same comments about using `get()` here. seems like both org and team are essential for this function to work, so if you access them with [slice] notation, you'll get well-formed errors

Reply:
Fixed when adding mypy